miso-web/src/main/webapp/scripts/list_poolelement.js

```diff
@@ -71,104 +71,107 @@ ListTarget.poolelement = {
-      "mData": "targetedSequencingId",
-      "include": Constants.isDetailedSample,
-      "mRender": ListUtils.render.textFromId(Constants.targetedSequencings, 'alias', '(None)'),
-      "iSortPriority": 0,
-      "bSortable": false
-    }, {
-      'sTitle': 'Indices',
-      'mData': 'indexIds',
-      'include': true,
-      'bSortable': false,
-      'iSortPriority': 0,
-      'mRender': function(data, type, full) {
-        var indices = Constants.indexFamilies.reduce(function(acc, family) {
-          return acc.concat(family.indices.filter(function(index) {
-            return data.indexOf(index.id) != -1;
-          }));
-        }, []).sort(function(a, b) {
-          return a.position - b.position;
-        });
+        }, {
+          "sTitle": "Targeted Sequencing",
+          "mData": "targetedSequencingId",
+          "include": Constants.isDetailedSample,
+          "mRender": ListUtils.render.textFromId(Constants.targetedSequencings, 'alias', '(None)'),
+          "iSortPriority": 0,
+          "bSortable": false
+        }, {
+          'sTitle': 'Indices',
+          'mData': 'indexIds',
+          'include': true,
+          'bSortable': false,
+          'iSortPriority': 0,
+          'mRender': function(data, type, full) {
+            var indices = Constants.indexFamilies.reduce(function(acc, family) {
+              return acc.concat(family.indices.filter(function(index) {
+                return data.indexOf(index.id) != -1;
+              }));
+            }, []).sort(function(a, b) {
+              return a.position - b.position;
+            });
 
-        var combined = indices.map(function(index) {
-          return index.sequence;
-        }).join('');
+            var combined = indices.map(function(index) {
+              return index.sequence;
+            }).join('');
 
-        var html = indices.map(function(index) {
-          return index.label;
-        }).join(', ');
+            var html = indices.map(function(index) {
+              return index.label;
+            }).join(', ');
 
-        return html;
-      }
-    }, {
-      'sTitle': 'Last Modified',
-      'mData': 'lastModified',
-      'include': true,
-      'iSortPriority': 0,
-      'sClass': 'noPrint'
-    }, {
-      'sTitle': 'Low Quality',
-      'bSortable': false,
-      'mData': 'library.lowQuality',
-      'mRender': function(data, type, full) {
-        return data ? "⚠" : "";
-      },
-      'include': true,
-      'iSortPriority': 0
-    }, {
-      "sTitle": "QC Passed",
-      "mData": "library.qcPassed",
-      "include": true,
-      "iSortPriority": 0,
-      "mRender": ListUtils.render.booleanChecks,
-      "bSortable": false
-    }];
+            return html;
+          }
+        }, {
+          'sTitle': 'Last Modified',
+          'mData': 'lastModified',
+          'include': true,
+          'iSortPriority': 0,
+          'sClass': 'noPrint'
+        }, {
+          'sTitle': 'Low Quality',
```

Comment:
What do you think about displaying this low quality flag in the Warnings column instead of its own column?

Reply:
Sure, will do